samcli/commands/deploy/command.py

```diff
@@ -172,12 +185,23 @@ def do_cli(
     use_json,
     tags,
     metadata,
+    interactive,
     region,
     profile,
 ):
     from samcli.commands.package.package_context import PackageContext
     from samcli.commands.deploy.deploy_context import DeployContext
 
+    confirm_changeset = False
```

Comment:
confirm changeset also needs to be its own flag in the CLI

Reply:
Good point. Will add